internal/ui/multiselect.go

```
@@ -396,24 +334,30 @@ func (m *Model) handleBatchProgress(msg batchProgressMsg) (tea.Model, tea.Cmd) {
-	b.WriteString("\n")
+	wrapW := m.width - 10
+	if wrapW > 80 {
+		wrapW = 80
+	}
+	if wrapW < 40 {
+		wrapW = 40
+	}
 
-	overlayHeight := 11
+	// --- Header (always visible) ---
+	header := StyleNormal.Render(fmt.Sprintf("%s %d packages?", strings.Title(batch.op), len(batch.ops)))
```

golangci-lint (2.11.4)

[error] SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead. 

(staticcheck)